### DISCUSSION TITLE
Number input field bugs

### DISCUSSION THREAD

**Discussion by AbbottMc** [Mojang/minecraft-editor#175] [Bugs]
number input field allow input letters, and once there got letters in the field, the selection cursor will be stuck in the starting position
![image](https://github.com/Mojang/minecraft-editor/assets/16969459/eaf448c3-5b17-419f-8f94-6f5e6d147a32)

**Comment by m-brisch** [comment]
@AbbottMc can you share more details on how you got into that state? I'm not able to reproduce the issue in-game

**Reply by AbbottMc** [reply]
just select the input field, make it be focused, and input letters, then you can see the letters be inputted successfully and the cursor being fixed in the start of letters.

BTW, I don't know if it matters but this bug was found in the field of custom editor extension, and the code is under here: 

```ts
  pane.addNumber(settings, 'randomHeightVariance', {
    titleStringId: 'sample.treegenerator.pane.variance',
    titleAltText: 'Tree Height Random Variance',
    min: 0,
    max: 5,
    showSlider: true,
  });
```
